draft-ietf-regext-rdap-geofeed.md

```diff
@@ -156,18 +156,19 @@ The following is an elided example of an IP network object with a geofeed link o
 
 # Operational Considerations
 
-When an RDAP server is queried for an IP network for a given address range, it is required to return the most-specific
-IP network object that covers the address range. That IP network object may not have an associated geofeed link, but it
-is possible that a less-specific IP network object does have such a link. Clients attempting to retrieve geofeed data
-for a given address range via RDAP should consider whether to retrieve the parent object for the initial response (and
-so on, recursively) in the event that the initial response does not contain geofeed data. Conversely, server operators
-should consider interface options for resource holders in order to support the provisioning of geofeed links for all
-networks covered by the associated data.
+When an RDAP server is queried for an IP network by a given IP address range, it is required to return the
```

Comment:
'by' makes it read as though the address of the client querying the server affects the result.  Suggested: "When an RDAP client performs an IP network lookup, per section 3.1.1 of RFC 9082, the RDAP server is required to return the most-specific IP network object that covers the IP address range provided by the client."

Reply:
Perfect.